### PR TITLE
Fix transition

### PR DESCRIPTION
**Pull request by posva**
Following work from @kingwl
#4900

### CODE REVIEW

File: test/unit/features/transition/transition-group.spec.js [hunk 1]
```diff
@@ -293,5 +293,44 @@ if (!isIE9) {
...
+            `<div class="test-empty-array-enter test-empty-array-enter-active">0</div>` +
+          `</span>`
+        )
+      }).thenWaitFor(duration / 3).then(() => {
+        vm.items.push(1)
+      }).thenWaitFor(nextFrame).then(() => {
+        expect(vm.$el.innerHTML).toBe(
+          `<span>` +
+            `<div class="test-empty-array-enter-active test-empty-array-enter-to">0</div>` +
+            `<div class="test-empty-array-enter-active test-empty-array-enter-to">1</div>` +
```
Comment (posva) [inline test/unit/features/transition/transition-group.spec.js]: I think the class should actually be `test-empty-array-enter test-empty-array-enter-active` 😕

**Comment by Kingwl** [comment]
sorry for the delay
the day before  yesterday is a Chinese festival: the Lantern Festival so I went to accompany my family
i think something is wrong and i'm try to find that

**Comment by Kingwl** [comment]
@posva 
hello i think i fix that
and i take a pr to your repo
please review that

File: test/unit/features/transition/transition-group.spec.js (hunk 1 above)
Comment (Kingwl) [inline test/unit/features/transition/transition-group.spec.js]: NextFrame looks nextTwoFrame

**Comment by posva** [comment]
@Kingwl ooh 😆 I wasn't expecting that!

> sorry for the delay
the day before yesterday is a Chinese festival: the Lantern Festival.
so I went to accompany my family

No need to apologise!

**Comment by yyx990803** [comment]
The root cause is in the move detection method, I think it's more appropriate to fix it there. But many thanks to both of you @posva and @Kingwl for looking into this!

**Pull request closed without merging**